src/jit/lsrabuild.cpp

```diff
@@ -2556,9 +2561,11 @@ void LinearScan::BuildDefsWithKills(GenTree* tree, int dstCount, regMaskTP dstCa
     VARSET_TP liveLargeVectors(VarSetOps::UninitVal());
     bool      doLargeVectorRestore = false;
 #endif // FEATURE_PARTIAL_SIMD_CALLEE_SAVE
+
+    buildKillPositionsForNode(tree, currentLoc + 1, killMask);
```

Comment:
ok, I see you've changed how `killMask` works slightly. It would be useful to add a comment about argument `killMask` both in `LinearScan::buildKillPositionsForNode` (which is already missing a comment), and in `LinearScan::BuildDefsWithKills`. In the latter case, it seems the existing comment "The mask of registers killed by this node" is insufficient if we allow `RBM_NONE` now.

Or if the callers passed in `RBM_INT_CALLEE_SAVED` like I mentioned above, would existing semantics (and comments about `killMask`) be preserved?

Reply:
The reason I did it this way is that this marker instruction selectively kills registers -- non GC refs aren't impacted. And it looks like `killMask` is more definitive and describes registers that must be considered dead no matter what. For instance any callee save mentioned there will be considered  as modified and so saved in the prolog.

Reply:
Makes sense. It would be useful to make sure it's clearly documented.

Reply:
I Agree with Bruce